qubes-rpc-policy/80-whonix.policy

```diff
@@ -1,11 +1,13 @@
-# service            arg source            target            action params
+# service            arg       source            target            action params
 
-whonix.SdwdateStatus  +  @tag:anon-gateway @tag:anon-vm      allow autostart=no notify=no
-whonix.SdwdateStatus  +  @tag:anon-gateway @default          deny notify=no
-whonix.SdwdateStatus  +  @anyvm            @anyvm            deny
+whonix.SdwdateStatus +         @tag:anon-gateway @tag:anon-vm      allow autostart=no notify=no
+whonix.SdwdateStatus +         @tag:anon-gateway @default          deny notify=no
+whonix.SdwdateStatus +         @anyvm            @anyvm            deny
 
-whonix.NewStatus      *  @tag:anon-vm      @tag:anon-gateway allow autostart=no
-whonix.NewStatus      *  @anyvm            @anyvm            deny
+whonix.NewStatus     *         @tag:anon-vm      @tag:anon-gateway allow autostart=no
+whonix.NewStatus     *         @anyvm            @anyvm            deny
```

Comment:
@marmarek is it best for whitespace changes like these to be a separate commit?  Mostly asking for myself in the future.

Reply:
I was also wondering if I should increase the whitespace. `whonix.GatewayCommand +restart` broke the spacing for the table. So it was either:

* keeping `whonix.GatewayCommand` lines longer, different from other line styles such as `whonix.NewStatus`, breaking the table versus
* a lot whitespace for lines that actually don't need it.

Or change whitespace during this pull request versus uneven whitespace depending on simple arg (`*` or `+` versus full keywords).

I'd hope git commit prettyness policy is relaxed/lenient. It's difficult for me to get the sources right. Pretty commit messages, squashing commits and all of that would make it an even greater hurdle for me. In result, it might lead to fewer contributions to cognitive load.

As for stylistic / whitespace improvement, if any, I'd appropriate adding git commits on top of what's already committed to the git branch here.